search/result.cpp

```diff
@@ -66,7 +66,7 @@ FeatureID const & Result::GetFeatureID() const
 {
 #if defined(DEBUG)
   auto const type = GetResultType();
-  ASSERT(type == RESULT_FEATURE, (type));
+  ASSERT(type == RESULT_FEATURE || type == RESULT_SUGGEST_FROM_FEATURE, (type));
```

Comment:
В search::Result хранится только один feature ID. Я не знаю, как это правильно отключить для интерактивного поиска, а Витя вообще не считает, что это нужно делать.

Кто прав?

> On Mar 21, 2016, at 18:56, ygorshenin <notifications@github.com> wrote:
> 
> In search/result.cpp <https://github.com/mapsme/omim/pull/2384#discussion_r56846563>:
> 
> > @@ -66,7 +66,7 @@ FeatureID const & Result::GetFeatureID() const
> >  {
> >  #if defined(DEBUG)
> >    auto const type = GetResultType();
> > -  ASSERT(type == RESULT_FEATURE, (type));
> > +  ASSERT(type == RESULT_FEATURE || type == RESULT_SUGGEST_FROM_FEATURE, (type));
> Точнее, несколько фич могут попадать под название что прописано в Suggest.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2384/files/0c5f2b528de8c080b03b97e0961cdefcc0f8af2f#r56846563>